Library/Homebrew/test/language/node_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "language/node"
+
+describe Language::Node do
+  specify "#npm_cache_config" do
+    shutup do
+      ret_val = described_class.npm_cache_config
+      expect(ret_val).to eq("cache=#{HOMEBREW_CACHE}/npm_cache\n")
+    end
+  end
+
+  describe "#pack_for_installation" do
+    it "raises error with non zero exitstatus" do
+      shutup do
+        expect { described_class.pack_for_installation }.to raise_error
+      end
+    end
+
+    it "does not raise error with a zero exitstatus" do
+      shutup do
+        allow_any_instance_of(Process::Status).to receive(:exitstatus).and_return(0)
+        expect { described_class.pack_for_installation }.not_to raise_error
+      end
+    end
+  end
+
+  describe "#setup_npm_environment" do
+    it "npmrc exists" do
+      shutup do
```

Comment:
Remove `shutup` unless output is created. Also add a `allow_any_instance_of(Pathname).to receive(:exist?).and_return(true)` to not cause a false-negative if the user does have that file when running `brew tests`